index.js

```diff
@@ -240,15 +239,6 @@ function loadConfig(program) {
       }
       config.UserAgent = `${pkg.name}/${pkg.version} ${distroName}`
 
-      if (process.env.LOGDNA_PLATFORM) {
```

Comment:
@darinspivey, people use `LOGDNA_PLATFORM` for some other platforms as well based on what I have seen in audits - it's not just for `k8s`

Reply:
ok.  I'll just remove the k8s logic then.